### PR TITLE
Stomach cancer concept model

### PR DESCRIPTION
**Pull request by Ninicorn**

### CODE REVIEW

**Review by yongqx — APPROVED** [review]

File: docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst [hunk 1]
```diff
@@ -441,8 +441,12 @@ We first need to obtain an atrophy state. To do that we give every simulant an a
 
 With the simulant's sex, age and atrophy propensity, use the tables above to figure out what atrophic state the propensity corresponds to and assign this to the simulant. If the propensity is < the proportion in the table, they are atrophic+. Update the simulant's atrophic state as they age through the simulation.   
 
+:underline:`B. Determine H. pylori and atrophy for simulants initalized into the pre-clinical state`
 
-:underline:`B. Obtain H. pylori status conditional upon age and atrophic state`
+  Assign all simulants initialized in the pre-clinical cancer state to atrophy positive.
```
Comment (aflaxman) [inline docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst:446]: This seems complicated, can I not do it, and instead we include as a limitation that we have not correlated the preclinical stomach cancer status and h. pylori/atrophy status?

File: docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst [hunk 2]
```diff
@@ -588,23 +594,24 @@ Stomach cancer screening algorithm was derived from the 2019 guidelines from the
...
 
+  - For **PC** population: All simulants initialized in the pre-clinical cancer state are assigned to be atrophy positive. Their H. pylori status is the fraction of H. pylori+ among atrophy (f_atrophy+) for that age group. When simulants attend their first ABC screening, PC cancers will be detected with 77% sensitivity. Those who are false negatives (23%) will be assigned into branch 1 (group A) or branch 2 (group B) depending on their H. pylori status for continued surveillance. (Their H. pylori status helps to determine their screening frequency in case of false negative.)
```
Comment (aflaxman) [inline docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst:614]: Can we use this sensitivity and specificity, but not correlate atrophy and pre-clinical cancer in the initialization?

Comment (aflaxman) [reply docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst:614]: will there be false positives?  what value should I use for specificity?

**Review by aflaxman — APPROVED** [review]

File: docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst (hunk 1 above)
Comment (Ninicorn) [reply docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst]: ok- I've gotten rid of this and added as a limitation.

File: docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst (hunk 2 above)
Comment (Ninicorn) [reply docs/source/concept_models/vivarium_swissre_stomachcancer/concept_model.rst]: we are using 98% as specificity as the combined specificity after endoscopy confirmation.

**Pull request merged**